templates/santa/participants.html.twig

```diff
@@ -152,8 +155,11 @@
                         html = 'No user selected';
                     }
 
-                    summaryCount.innerHTML = users.length + ' people';
-                    checkedUsersSummary.innerHTML = html;
+                    summaryCount.innerHTML = users.length + (users.length <= 1 ?' person' : ' people');
+                    checkedUsersSummary.innerHTML = html
+                    html = checkedUsersSummary.innerText
+                    checkedUsersSummary.innerHTML = html
```

Comment:
This looks a bit ugly :sweat_smile: I think your encoding problem can be due to an extra usage of `|escape('html_attr')` , as explained above :wink:

Reply:
I tried removing it as suggested, but as said before it doesn't really work out. I do not have any prettier way of doing this (surely just because I'm not seeing the obvious solution), so I will be leaving it for now